terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -52,6 +52,7 @@ module "glue_reporting_hub_job" {
     "--enable-spark-ui"                         = false
     "--enable-job-insights"                     = true
     "--dpr.aws.kinesis.endpointUrl"             = "https://kinesis.${local.account_region}.amazonaws.com"
+    "--dpr.contract.registryName"               = "${module.glue_registry_avro.name}"
```

Comment:
This will fail, you may need below inside the module outputs glue_registry

output "regitry_name" {
  description = "Registry Name"
  value       = aws_glue_registry.glue_registry[0].id
}

Reply:
Ok, will review other modules and add the necessary properties. Missed this (assumed it'd just work ;) ).

Reply:
Changes pushed. Waiting for checks.